content/en/docs/instrumentation/cpp/getting-started-new.md

```diff
@@ -0,0 +1,294 @@
+- [Conan](https://conan.io/)
+
+## Example Application 
+
+The following example uses a basic [Oat++](https://oatpp.io/) application. If you are not using 
+Oat++, that's OK - you can use OpenTelemetry C++ with other web frameworks 
+as well, such as Crow and POCO. For a complete list of 
+libraries for supported frameworks, see the [registry](/ecosystem/registry/?language=cpp).  
+
+## Dependencies 
+
+To begin, set up an environment in a new directory called `cpp-simple`. Within
+that directory, install the oatpp locally using package manager Conan.
+
+Create a Conan profile for your project 
+
+```bash
+conan profile new cpp-simple --detect
+```
+
+Install oatpp in your project directory using Conan 
+
+```bash
+conan install . --profile=cpp-simple 
+```
+
+This will create a `conanfile.txt` file in your project directory, which includes oatpp as a dependency.
+
+Now open the `conanfile.txt` file in a text editor and add oatpp as a dependency:
+
+```plaintext
+[requires]
+oatpp/1.2.0  # Use the desired version of oatpp
+
+[generators]
+cmake
+```
+
+Finally run the following command to locally install oatpp the project dependency: 
+
+```bash
+conan install . 
+```
+
+## Create and launch an HTTP Server
+
+In that same folder, create a file called `main.cpp` and add the following code to the file:
+
+```cpp
+#include "oatpp/web/server/HttpConnectionHandler.hpp"
+#include "oatpp/network/Server.hpp"
+#include "oatpp/network/tcp/server/ConnectionProvider.hpp"
+#include <cstdlib>
+#include <ctime>
+#include <string>
+
+using namespace std;
+
+
+class Handler : public oatpp::web::server::HttpRequestHandler {
+public:
+  shared_ptr<OutgoingResponse> handle(const shared_ptr<IncomingRequest>& request) override {
+    int low = 1;
+    int high = 7;
+    srand((int)time(0));
+    int random = rand() % (high - low) + low;
+    // Convert a std::string to oatpp::String
+    const string response = to_string(random); 
+    return ResponseFactory::createResponse(Status::CODE_200, response.c_str());
+  }
+};
+
+void run() {
+  auto router = oatpp::web::server::HttpRouter::createShared();
+  router->route("GET", "/rolldice", std::make_shared<Handler>());
+  auto connectionHandler = oatpp::web::server::HttpConnectionHandler::createShared(router);
+  auto connectionProvider = oatpp::network::tcp::server::ConnectionProvider::createShared({"localhost", 8000, oatpp::network::Address::IP_4});
+  oatpp::network::Server server(connectionProvider, connectionHandler);
+  OATPP_LOGI("MyApp", "Server running on port %s", connectionProvider->getProperty("port").getData());
+  server.run();
+}
+
+int main() {
+  oatpp::base::Environment::init();
+  run();
+  oatpp::base::Environment::destroy();
+  return 0;
+}
+
+```
+
+Create another file called `CMakeLists.txt` and it is used to define the build process for 
+your C++ project, inclduing how to compile and link your oatpp and OpenTelemetry 
+application. Here is what `CMakeLists.txt` might look like:
```

Comment:
```suggestion
Create another file called `CMakeLists.txt` and it is used to define the build process for 
your C++ project, inclduing how to compile and link your oatpp
application. Here is what `CMakeLists.txt` might look like:
```

In this step we are not adding OpenTelemetry yet, it should just make sure that the app can be run as-is